calendar-bundle/src/Resources/contao/dca/tl_calendar_events.php

```diff
@@ -1329,4 +1329,17 @@ public function toggleVisibility($intId, $blnVisible, DataContainer $dc=null)
 			$dc->invalidateCacheTags();
 		}
 	}
+
+	/**
+	 * @param DataContainer $dc
+	 *
+	 * @return array
+	 */
+	public function addSitemapCacheInvalidationTag($dc, array $tags)
+	{
+		$calendar = CalendarModel::findByPk($dc->activeRecord->pid);
+		$pageModel = PageModel::findWithDetails($calendar->jumpTo);
```

Comment:
I think we should handle the `!$pageModel` case, because the jumpTo ID can be invalid.

Reply:
I've added it to all callbacks. Even the ones where I think it should never happen but better safe than sorry :)